tests/ci/common_posix_setup.sh

```diff
@@ -61,6 +68,38 @@ function build_and_test {
   run_cmake_custom_target 'run_tests'
 }
 
+function generate_symbols_file {
+  # read_symbols.go currently only support static libraries
+  if [ ! -f  "$BUILD_ROOT"/crypto/libcrypto.a ]; then
+    echo "Static library not found: ${BUILD_ROOT}/crypto/libcrypto.a"
+    print_system_and_dependency_information
+    exit 1
+  fi
+
+  go run "$SRC_ROOT"/util/read_symbols.go -out "$BUILD_ROOT"/symbols_crypto.txt "$BUILD_ROOT"/crypto/libcrypto.a
+  go run "$SRC_ROOT"/util/read_symbols.go -out "$BUILD_ROOT"/symbols_ssl.txt "$BUILD_ROOT"/ssl/libssl.a
+
+  # The $BUILD_ROOT gets deleted on each run. symbols.txt must be placed elsewhere.
+  cat "$BUILD_ROOT"/symbols_crypto.txt  "$BUILD_ROOT"/symbols_ssl.txt | grep -v -e '^_\?bignum' >  "$SRC_ROOT"/symbols.txt
+}
+
+
+function verify_symbols_prefixed {
+  nm -g "$BUILD_ROOT"/crypto/libcrypto.a | grep "aws_lc_1_1_0"
+  nm -g "$BUILD_ROOT"/ssl/libssl.a | grep "aws_lc_1_1_0"
+}
```

Comment:
Nit: this verification may only indicate the `aws_lc_1_1_0` exists. May we can have a `check_symbols.go` by extending the code of `read_symbols.go` to ensure each symbol this prefixed? My comment is a little over and should not block this PR.

Reply:
We could probably just re-use `read_symbols.go` to verify this.  I'll try updating the script to do that instead.

Reply:
I just point out this but I actually don't know how the verification can be enhanced.

Reply:
I was going to ask something similar to this. Is there a way to verify that the symbols have been correctly prefixed? Maybe a script that checks if the "to-prefix" symbol list generated in `symbols_crypto.txt` and `symbols_ssl.txt` align with the prefixed symbols in `libcrypto.a` and `libssl.a`?